### ISSUE TITLE
Unterstützung mehrerer Aktenkreise

### ISSUE THREAD

**Issue by Mucki526** [jlawyerorg/j-lawyer-org#241]
Ich führe Akten in verschiedenen Adressbereichen z.B. Patentsachen ab 30000 ff. Es wäre schön, wenn man bei Neuanlage einer Akte Adressbereiche in einer Auswahl (z.B. Dropdownmenue) anlegen und dann die nächste freie Akte ab der jeweiligen Auswahl anlegen könnte. (also Dropdown 30.000 ff. -> nächste freie Akte ist dann 30006. Im Moment wähle ich jedesmal manuell eine Adressbereich aus und lege die nächste freie Akte an.....

**Comment by j-dimension** [comment]
Wo legst Du diesen Wert denn aktuell ab? In den "eigenen" Feldern?

**Comment by Mucki526** [comment]
Nein, ich ändere die Startnummer unter Einstellungen / Akten / Aktenzeichenschema, Benutze Schema in der Version unter Punkt 2 von oben. Standard ist zur Zeit. z.B. 600/2019 für Kanzleiakten Standard, Kanzlei Organisation 20000/2019, 30000 für Patentsachen, u.s.w. und lege dann meine neue Akte an. Dann stelle ich den Standard wieder ein... Umständlich aber es geht. Rückwärts habe ich einmalig die Altakten angelegt und die Jahreszahlen, die ja leider nur das aktuelle Jahr haben der Aktenanlage über setrefno korrigiert bzw. bin da noch dabei... Auf dieses Weise habe ich das richtige Jahr und den richtigen Nummernbereich je nach Verwendung.

**Comment by j-dimension** [comment]
Danke, jetzt verstehe ich was gewünscht ist. Ich erlaube mir das Ticket etwas umzuformulieren.

**Comment by j-dimension** [comment]
Unterstützung mehrerer Aktenkreise

- in einer Datenbank kann es mehrere Aktenkreise mit unterschiedlichen Schemata geben
- es kann mehrere Aktenkreise mit identischem Schema aber unterschiedlichem Minimalwert für lfd. Nummer geben
- sobald >1 Aktenkreis definiert ist, muss bei Aktenneuanlage ein Aktenkreis ausgewählt werden, sobald eine Speicherung stattfindet.
